doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc

```diff
@@ -51,10 +51,10 @@ end
 ----
 +
 This business rule detects when a file named `invalid.pdf` is uploaded to the case file. It then removes the `invalid.pdf` document and creates a new instance of the `Prepare hardware spec` user task.
-. Click *Build*, then *Deploy* to deploy the IT Orders project.
+. Click *Build*, then *Deploy* to deploy the *IT_Orders* project.
 . Create a file called `invalid.pdf` and save it locally.
 . Create a file called `valid-spec.pdf`and save it locally.
-. Open the IT Orders project in {CENTRAL} under *Menu* -> *Projects* -> *IT Orders*.
+. Open the *IT_Orders* project in {CENTRAL} under *Menu* -> *Projects* -> *IT_Orders*.
```

Comment:
I would rephrase something like:
In {CENTRAL}, go  to *Menu* -> *Projects* -> *IT_Orders*  and open the IT ORDERS project.